fuxi-reward-claims/sig-files/slamper-3CDE464E307C0C304F48214DCB0F38C5C213D585.json

```diff
@@ -1,22 +1,24 @@
 -----BEGIN PGP SIGNED MESSAGE-----
 Hash: SHA512
 
-iaa1a2havy4hctaq85nh6f9m3y4m9m36g6wcppdhhw
+faa19cqf63aakkx82eylzv7n04n0g3vezk827yksan
+
+Pubkey: fap1addwnpepqtwqgzxledvgv3r4j7n6zgzu5g8dfh085tjuu65202xm8yyzke4ukrz3mng
```

Comment:
Hi Hendrik! You should use/build the `mainnet` binary to create addresses that start with `i` instead of `f`.

Reply:
Oh that's thanks for catching that. We messed that up in the 2nd run.

I will change the prefix and push a fix